test/e2e_node/restart_test.go

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2015 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e_node
+
+import (
+	"k8s.io/kubernetes/test/e2e/framework"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	"k8s.io/kubernetes/pkg/api"
+	"os/exec"
+)
+
+var _ = framework.KubeDescribe("Restart [Serial] [Slow] [Disruptive]", func() {
```

Comment:
Oh yeah, node e2e has the default request so I guess 100 1m pods will fit. 
Still, probably better to make it run on different workers.